lsp-lens.el

```diff
@@ -82,18 +89,21 @@ Results are meaningful only if FROM and TO are on the same line."
                                          (beginning-of-line-text)
                                          (point))))
          (str (concat (make-string offset ?\s)
```

Comment:
Hm, I think that offset should not be used when lens type = end-of-line

Reply:
I didn't notice any changes, but I agree it's unnecessary, done!

Reply:
It will make a difference when the offset is != 0.